tests/3_docker_daemon_configuration_files.sh

```diff
@@ -225,7 +225,7 @@ check_3_8() {
     fail=0
     perms=$(find "$directory" -type f -name '*.crt')
     for p in $perms; do
-      if [ "$(stat -c %a $p)" -ne 444 ] && [ "$(stat -c %a $p)" -ne 400 ]; then
+      if [ "$(stat -c %a $p)" -gt 444 ]; then
```

Comment:
`-gt`

Reply:
Yeah this check uses slightly different logic than the others - if it is true it fails, not passes. It used to be:

`      if [ "$(stat -c %a $p)" -ne 444 ] && [ "$(stat -c %a $p)" -ne 400 ]; then
        fail=1
      fi`

So I've kept the same logic using -gt instead of double -ne.